rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cat.health/10_basic.yml

```diff
@@ -71,9 +91,96 @@
                 \d+            \s+ # relo
                 \d+            \s+ # init
                 \d+            \s+ # unassign
+                \d+            \s+ # unassign.pri
                 \d+            \s+ # pending_tasks
                 (-|\d+(?:[.]\d+)?m?s) \s+ # max task waiting time
                 \d+\.\d+%             # active shards percent
                 \n
               )+
             $/
+
+---
+"Help - Without unassigned_pri_shard_count":
+  - skip:
+      cluster_features: gte_v8.15.0
+      reason: Capabilities skipping does not yet work in mixed clusters. This will disable the test in such scenarios.
```

Comment:
Are these tests ever executed? According to the docs:
```
Note that `skip` with `capabilities` or `cluster_features` will skip the test if *any* node in the cluster
has the feature or capability. `requires` will only run the test if *all* of the nodes in the cluster
have the feature or capability.
```
I understand this in the following way:
- Mixed clusters with the current version will be skipped.
- This would only run if the all nodes of the cluster are on version less than 8.15.0

Considering the above I am not sure that these test are worth having. Clusters with mixed versions less than 8.15.0 should have been tested by their own tests. I would like to have mixed cluster tests but I would like one of the nodes to be on the current version.

What do you think?

Reply:
I think you are right... With the capabilities API I was intending to test the case of a mixed cluster when you hit an older node as a coordinator but as that API does not work correctly in this case and we had to switch to the features api, I think the test is now redundant.

Reply:
Tests removed in latest commit